hibernate-reactive-core/src/main/java/org/hibernate/reactive/boot/dialect/ReactivePostgresDialect.java

```diff
@@ -0,0 +1,31 @@
+package org.hibernate.reactive.boot.dialect;
+
+import org.hibernate.dialect.PostgreSQL10Dialect;
+import org.hibernate.type.descriptor.sql.SqlTypeDescriptor;
+import org.hibernate.type.descriptor.sql.VarbinaryTypeDescriptor;
+import org.hibernate.type.descriptor.sql.VarcharTypeDescriptor;
+
+import java.sql.Types;
+
+/**
+ * A {@link org.hibernate.dialect.Dialect} for PostgreSQL 10 and
+ * above that customizes the handling of LOB types for use with
+ * the Vert.x {@link io.vertx.sqlclient.SqlClient}.
+ */
+public class ReactivePostgresDialect extends PostgreSQL10Dialect {
+	public ReactivePostgresDialect() {
+		super();
+		registerColumnType( Types.BLOB, "bytea" );
+	}
+	@Override
+	public SqlTypeDescriptor getSqlTypeDescriptorOverride(int sqlCode) {
+		switch ( sqlCode ) {
+			case Types.BLOB:
+				return VarbinaryTypeDescriptor.INSTANCE;
+			case Types.CLOB:
+				return VarcharTypeDescriptor.INSTANCE;
+			default:
+				return super.getSqlTypeDescriptorOverride( sqlCode );
+		}
+	}
```

Comment:
@Sanne can you think of any other way to override the type mappings for `BLOB` and `CLOB` that doesn't involve subclassing the `Dialect`s?

(Bearing in mind that the dialects already override these mappings.)

I could not find, and can't recall, any way to do this globally.